visearch-android/src/main/java/com/visenze/visearch/android/model/ProductResult.java

```diff
@@ -0,0 +1,115 @@
+package com.visenze.visearch.android.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Map;
+
+public class ProductResult {
```

Comment:
rename to Product

Reply:
in future, would be nice to have wrapper methods here for various field types e.g. getCurrencyFieldValue(String fieldName) --> CurrencyField (price, currency)

Reply:
changed already